src/component/Board.jsx

```diff
@@ -101,21 +119,28 @@ function Board() {
 
   return (
     <>
-      <header>
+      <header className="board-header">
+        <h2 className="board-heading score">Score: {pairCounter}</h2>
```

Comment:
You aren't updating `pairCounter` value anywhere

Reply:
i am updating pairCounter on Match found, and it's working smoothly on UI as well.